### ISSUE TITLE
Unable to install `com.cnuernber/ham-fisted` with Neil 0.1.59

### ISSUE THREAD

**Issue by teodorlu** [babashka/neil#177]
I'm seeing this CLI behavior:

```
$ ls
$ neil dep search ham-fisted
:lib com.cnuernber/ham-fisted :version "1.000-beta-90" :description nil
$ neil dep add com.cnuernber/ham-fisted
$ ls
deps.edn
$ cat deps.edn
{:deps {}
 :aliases {}}
```

I'd expect `neil dep add` to add the dependency when we can find it with `neil dep search`.

**Comment by teodorlu** [comment]
From @borkdude on Slack:

> I suspect it has to do with neil not adding non-stable versions by default
> but if you specify the `:version` yourself, then it will work

Teodor:

> ahh, you're right. setting version explicitly adds everything as expected. `neil dep search ham-fisted | fzf | xargs neil dep add` also just works.
> Any idea on what kind of semantics we want here?
> "Do nothing, print nothing and return `:exit 0`" kind of surprised me.

**Comment by teodorlu** [comment]
Fixed with #178 , closing.
